### PR TITLE
LPS-158194 Adjust test coverage for walkthrough popover end process

### PR DESCRIPTION
**Pull request by john-co**
**Description**
Behavior of closing the last step of a walkthrough popover [has been updated to reset the the process](https://github.com/liferay-frontend/liferay-portal/pull/2505). Adjust the test coverage to account for this.

**Test Plan**
- [x] Walkthrough popover test passes

**JIRA TICKET**
https://issues.liferay.com/browse/LPS-158194

### CODE REVIEW

**Comment by john-co** [comment]
ci:test:relevant

**Comment by john-co** [comment]
ci:test:sf

File: modules/apps/frontend-js/frontend-js-test/src/testFunctional/tests/WalkthroughPopover.testcase [hunk 1]
```diff
@@ -204,7 +204,7 @@ definition {
 
 	@description = "LPS-154648. Assert close button on final step can end walkthrough process."
```
Comment (susanchen3) [inline modules/apps/frontend-js/frontend-js-test/src/testFunctional/tests/WalkthroughPopover.testcase:205]: Need to update the description to reset walkthrough process

File: modules/apps/frontend-js/frontend-js-test/src/testFunctional/tests/WalkthroughPopover.testcase [hunk 2]
```diff
@@ -227,13 +227,17 @@ definition {
...
 
-		task ("Then hotspot element is not present") {
-			AssertElementNotPresent(locator1 = "Walkthrough#HOTSPOT");
-		}
-
 		task ("And Then popover is not present") {
 			AssertElementNotPresent(locator1 = "Walkthrough#POPOVER");
 		}
+
+		task ("Then hotspot element is present on 1st element") {
```
Comment (susanchen3) [inline modules/apps/frontend-js/frontend-js-test/src/testFunctional/tests/WalkthroughPopover.testcase:234]: Nitpick: this task would be "And Then ..."

File: modules/apps/frontend-js/frontend-js-test/src/testFunctional/tests/WalkthroughPopover.testcase [hunk 3]
```diff
@@ -227,13 +227,17 @@ definition {
 			Click(locator1 = "Button#CLOSE");
 		}
 
-		task ("Then hotspot element is not present") {
-			AssertElementNotPresent(locator1 = "Walkthrough#HOTSPOT");
-		}
-
 		task ("And Then popover is not present") {
```
Comment (susanchen3) [inline modules/apps/frontend-js/frontend-js-test/src/testFunctional/tests/WalkthroughPopover.testcase:230]: Nitpick: this task would be "Then ..."

**Review by susanchen3 — APPROVED** [review]
Just some minor description fixes :) I see why the test didn't on Diego's PR now :thinking:

**Comment by susanchen3** [comment]
ci:forward

**Pull request closed without merging**